cloudformation/compute-fleet-hit-substack.cfn.yaml

```diff
@@ -292,6 +299,7 @@ Resources:
                 echo "Reporting instance as unhealthy and dumping logs to ${!log_dir}/${!instance_id}.tar.gz"
                 tar -czf ${!log_dir}/${!instance_id}.tar.gz /var/log
                 # TODO: add possibility to disable this behavior
+                sleep 10
```

Comment:
can we add this sleep also to the compute fleet substack and add a small comment saying we do this to allow logs to be sent to CloudWatch?